frontend/assets/translations/es.json

```diff
@@ -0,0 +1,917 @@
+{
+    "globals": {
+        "readDocumentation": "Leer la documentación",
+        "cancel": "Cancelar",
+        "save": "Guardar",
+        "back": "Atrás",
+        "edit": "Editar",
+        "search": "Buscar",
+        "update": "Actualizar",
+        "delete": "Borrar",
+        "add": "Añadir",
+        "view": "Ver",
+        "create": "Crear",
+        "enabled": "Activado",
+        "disabled": "Desactivado",
+        "yes": "Si",
+        "submit": "Enviar",
+        "select": "Seleccionar",
+        "environmentVar": "Variables de entorno",
+        "saving": "Guardando...",
+        "saveDatasource": "Guardar fuente de datos",
+        "authorize": "Autorizar",
+        "connect": "Conectar",
+        "reconnect": "Reconectar",
+        "components": "componentes",
+        "send": "Enviar",
+        "noConnection": "no se puede conectar",
+        "connectionVerifeid": "conexión verificada",
+        "left": "Izquierda",
+        "center": "Centro",
+        "right": "Derechar",
```

Comment:
Hi, here is a small mistake, it should be `Derecha` without the last 'r'

Reply:
hell, yes. A big typo. Thanks!